Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM openjdk:17
```

Comment:
Maybe take a look at the gradle plugin called 'jib' for building docker images.

Reply:
Sure, will do.